.cargo/config.toml

```diff
@@ -5,7 +5,8 @@ rustflags = [
     # BEGIN - Embark standard lints v6 for Rust 1.55+
     # do not change or add/remove here, but one can add exceptions after this section
     # for more info see: <https://github.com/EmbarkStudios/rust-ecosystem/issues/59>
-    "-Dunsafe_code",
+    # TODO: Luc: How do I add exceptions?
+    # "-Dunsafe_code",
```

Comment:
Not sure how to add an exception here.

Reply:
I think you can do `#[allow(unsafe_code)]` where you need it.

Reply:
I think we can revert this and use the allow everywhere if still needed?